### PR TITLE
Limit aliases to formulary

### PR DESCRIPTION
**Pull request by StanczakDominik**
When we first discussed aliases in #855 and #856, I thought that we'd add them for the formulary only. In https://github.com/PlasmaPy/PlasmaPy/issues/856#issue-639988545, Erik wrote (emphasis mine):

> Explicit function names are great for readability and clarity, and the **primary functions should remain like that**. However, explicit names can be long and become burdensome for users. Users either work with the long names or have to individually import each function and rename it to a more manageable name. Thus, I propose we make **aliases for, at least, the most common formulary functionality**.

But while working on https://github.com/PlasmaPy/PlasmaPy/pull/1099 I found out about [`analysis.find_vf_`](https://docs.plasmapy.org/en/stable/api/plasmapy.analysis.swept_langmuir.find_vf_.html#plasmapy.analysis.swept_langmuir.find_vf_) and [`dispersion.tfds_`](https://docs.plasmapy.org/en/stable/api/plasmapy.dispersion.tfds_.html#plasmapy.dispersion.tfds_).

### Why I like formulary aliases and dislike them elsewhere

When working with the Formulary, we conceptually dive into the space of plasma physics concepts. It is reasonable to use a name such as `wp_` for the plasma frequency ω<sub>p</sub> because we're already in the mind state of expecting those. These expressions are likely to repeat in multiple other equations. https://github.com/PlasmaPy/PlasmaPy/pull/1059 or other equation-heavy codes are examples of a place where I might want to use the aliases. Reading and writing code like this, under such conditions, can be easier than with the full and explicit names. 
But these conditions are not fulfilled for general, "primary" functions, as noted by Erik. Neither `analysis` (yet?) nor `dispersion` induce a headspace of associations; and I don't think there is a potential context that induces you to say *"yes, it is obvious at first glance what tfds is"*.

I also suspect that these two functions would be unlikely to repeat in code as densely as formulary functions - sure, you might loop over parameters or datasets with them, but it's going to be rare to have multiple occurrences side by side.

Also, code is read more often than it is written; and ― this may not be codified anywhere, but I always thought one of the principles of our little project here was to promote people writing clean and readable research code. By making those aliases there, we directly promote their usage; and this works against that goal. People can, of course, define their own aliases; but I'd rather we didn't do much to promote it unless we have clear reasons to.

To sum up - I don't think it was the right call to introduce them. I thus propose their removal, and that we limit ourselves to the formulary from now on with officially sanctioned aliases.

The removal part is, of course, a breaking change; but I think it's better to cause their direct users some annoyance now rather than to doom generations of grad students to come to scratch their heads at what the heck `tfds` is... forever. This is, of course, sarcastic, but also not really. :wink:

### CODE REVIEW

**Comment by rocco8773** [comment]
Ok, I can get behind this argument, especially for `tfds_`.  However, `vf` (floating potential), `vp` (plasma potential), `isat` (ion-saturation current), etc. are very common nomenclatures in the Langmuir analysis world.  They are used extensively in literature and are usually used in place of their long forms.  So, I feel these are still appropriate because they still satisfy the statement, "It is reasonable to use a name such as `wp_` for the plasma frequency ωp because we're already in the mind state of expecting those."

With that said, I'm okay with putting a restriction on aliases such that aliases are only given for functionality that could see frequent use and the aliases names can only use shorthand nomenclature associated with that field/context.

**Comment by StanczakDominik** [comment]
Okay, I'm obviously a foreigner to that world, so I'll take your word on that. I'll revert the langmuir changes. Does that sound good?

File: changelog/1101.breaking.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Remove alias: `tfds` to `~plasmapy.dispersion.two_fluid_dispersion_solution`,
```
Comment (rocco8773) [inline changelog/1101.breaking.rst:1]:
```suggestion
Remove alias: `tfds_` to `~plasmapy.dispersion.two_fluid_dispersion_solution`,
```

**Review by rocco8773 — APPROVED** [review]
Just that one typo in the change log, otherwise looks good to me.
